### PR TITLE
27 tk group js and objects

### PR DESCRIPTION
**Pull request by TobiasKjernell**
Added object function for team members. Added pictures and information for each person for further tasks.

### CODE REVIEW

File: images/team/srilatha.jpg
Comment (jasminedns) [inline images/team/srilatha.jpg:1]: The image is too small compared to the others

File: scripts/aboutus.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    allTeamMembers.push(this);
+}
+
+const createMembers = () => {
+    tobiasK = new TeamMember("Tobias Kjernell", ["Music", "Spicy food", "Movies and TV-series", "Long naps"], "Sweden", ["Rabbit", "Cat", "Dog", "Squirrel", "Dolphin"],"../images/team/tobias.jpg");
+    srilathaP = new TeamMember("Srilatha Potnuru", ["Badminton"], "India", ["Dog", "Parrot", "Cat", "Peacock", "Panda"], "../images/team/srilatha.png");
+    carolineL = new TeamMember("Caroline Lindbom", ["Hiking"], "Sweden", ["Dog", "Alpacka", "Raven", "Owl", "Cat"], "../images/team/caroline.jpg");
+    mariymA = new TeamMember("Mariym Aqbal", ["Reading", "Watching historical films", "Music", "Cricket", "Tennis", "Football", "Hockey"], "Pakistan", ["Cat", "Rabbit", "Panda", "Kangaroo", "Pigeon"], "../images/team/mariym.png");
+    jasminD = new TeamMember("Jasmine Danese", ["Gaming", "Procrastinate"], "Italy", ["Otter", "Quokka", "Capybara", "Raccoon", "Fox"], "../images/team/jasmine.jpg");
```
Comment (jasminedns) [inline scripts/aboutus.js]: misspelled jasmine in L2 and L19

File: scripts/aboutus.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    this.topFiveAnimals = topFiveAnimals;
+    this.picture = picture;
+
+    allTeamMembers.push(this);
+}
+
+const createMembers = () => {
+    tobiasK = new TeamMember("Tobias Kjernell", ["Music", "Spicy food", "Movies and TV-series", "Long naps"], "Sweden", ["Rabbit", "Cat", "Dog", "Squirrel", "Dolphin"],"../images/team/tobias.jpg");
+    srilathaP = new TeamMember("Srilatha Potnuru", ["Badminton"], "India", ["Dog", "Parrot", "Cat", "Peacock", "Panda"], "../images/team/srilatha.png");
+    carolineL = new TeamMember("Caroline Lindbom", ["Hiking"], "Sweden", ["Dog", "Alpacka", "Raven", "Owl", "Cat"], "../images/team/caroline.jpg");
```
Comment (jasminedns) [inline scripts/aboutus.js]: misspelled alpaca

**Review by jasminedns — CHANGES_REQUESTED** [review]

**Review by potnurusrilatha — APPROVED** [review]
looks ok

**Review by Moon2024-light — APPROVED** [review]
looks good!

**Review by idacarlin95 — APPROVED** [review]
looks good to me

**Review by jasminedns — APPROVED** [review]
Good to go!

**Pull request merged**